README.md

```diff
@@ -585,7 +585,21 @@ For example, if the the developer would like to attach a custom `onLoad` callbac
 
 #### Lazy Loading
 
-If you'd like to lazy load images, we recommend using [lazysizes](https://github.com/aFarkas/lazysizes). In order to use react-imgix with lazysizes, you can simply tell it to generate lazysizes-compatible attributes instead of the standard `src`, `srcset`, and `sizes` by changing some configuration settings:
+If you'd like to lazy load images, we recommend using browser-level lazy loading, with the [`loading`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading) property, passed in `htmlAttributes`:
+
+```jsx
+<Imgix
+  src="..."
+  sizes="..."
+  htmlAttributes={{
+    loading: "lazy"
+  }}
+/>
+```
+
+This property has [strong browser support](https://caniuse.com/loading-lazy-attr), and functions without additional JavaScript. Additionally, using browser-level lazy loading enables optimization of the sizes attribute with `sizes="auto"`, which allows the browser to automatically calculate the optimal size for the image based on its layout. 
+
+If you need granular control over lazy-loading behavior such as loading distance, the Imgix component can also be used with [lazysizes](https://github.com/aFarkas/lazysizes). In order to use react-imgix with lazysizes, you can simply tell it to generate lazysizes-compatible attributes instead of the standard `src`, `srcset`, and `sizes` by changing some configuration settings:
```

Comment:
Another option to consider: recommending that users use the IntersectionObserver API if they would like more granular control.

Do you think it's just easier to continue recommending lazysizes instead of showing a little snippet demonstrating how to use IO with Imgix?

Reply:
The only reason I didn't suggest removing it in the first place is that the equivalent intersectionObserver code is a little longer/more complex, and where you initialize the observer kind of depends on the architecture of your app. @luqven if you're ok with not having a code snippet for it, we could just link to the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API)?

Reply:
I can see how the IO code example inline might not be as helpful as we'd like.

Maybe the middle ground here is to link to the MDN docs first like you suggest and still leave the reference to `lazysizes`.

suggestion:

```MD
If you need granular control over lazy-loading behavior such as loading distance, you can use the [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API).

If you are using a library like [lazysizes](https://github.com/aFarkas/lazysizes), you can tell the Imgix component to generate compatible attributes instead of the standard `src`, `srcset`, and `sizes` by changing some configuration settings:
```

This way we don't direct people to `lazysizes` but instead make it clear we support that use case.

Would that address everyone's concerns with this docs change?

Reply:
Sounds like a good compromise. I updated the PR to use that wording.